presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -211,10 +212,28 @@ public void checkCanShowColumns(ConnectorSecurityContext context, SchemaTableNam
     @Override
     public List<ColumnMetadata> filterColumns(ConnectorSecurityContext context, SchemaTableName tableName, List<ColumnMetadata> columns)
     {
-        if (!checkAnyTablePermission(context, tableName)) {
+        if (INFORMATION_SCHEMA_NAME.equals(tableName.getSchemaName())) {
```

Comment:
It would be nice to have some test for this special case.

Reply:
This special case is automatically attached to every rule in file-based access controls:  https://github.com/prestosql/presto/blob/master/presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedSystemAccessControl.java#L784-L786